insanely_fast_whisper_rocm/core/integrations/diarization.py

```
@@ -0,0 +1,345 @@
+
+    Returns:
+        The result dict with ``speaker`` added to each chunk and
+        ``diarized`` set to ``True``.  If pyannote is not installed, the
+        original result is returned unchanged.
+
+    Raises:
+        DiarizationError: If the HuggingFace token is missing or the model
+            cannot be loaded.
+    """
+    if Pipeline is None:
+        logger.warning("pyannote.audio is not installed – returning result unchanged")
```

Ruff (0.15.11)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)